h2-storage/src/main/java/com/cgi/hexagon/h2storage/order/AssemblyOrder.java

```diff
@@ -1,46 +1,47 @@
 package com.cgi.hexagon.h2storage.order;
 
 import com.cgi.hexagon.businessrules.Status;
-import com.cgi.hexagon.h2storage.user.AssemblyUser;
 import org.springframework.data.annotation.Id;
 
 import javax.persistence.*;
-import java.util.Date;
+import java.io.Serializable;
+import java.time.LocalDateTime;
 
 @Entity
-public class AssemblyOrder{
+@Table(name = "assembly_order")
+public class AssemblyOrder implements Serializable {
 
     @Id
     @GeneratedValue
-    long id;
+    @Column(name = "id")
+    private long id;
 
-    String goodsId;
+    @Column(name = "shop_order_id")
```

Comment:
do we need all these annotations? why?

Reply:
It is wrong...
goodsId is the productId at the shop
shopId is the id of the order at the shop